coding/python_testing.rst

```diff
@@ -42,44 +41,134 @@ The important things to note in this example are:
 * When testing that an exception is raised always use :meth:`~unittest.TestCase.assertRaises` as a context manager, as shown in line 10 of the above example.
 * If a test method completes, the test passes; if it throws an uncaught exception the test has failed.
 
-Using a more advanced test runner
-=================================
+Supporting Pytest
+=================
 
 .. note::
-   `pytest`_ is not currently installed as part of a LSST stack installation.
-   It can be installed using :command:`conda install pytest` or :command:`pip install pytest`.
+  `pytest`_ and its plugins are standard stack EUPS packages and do not have to be installed separately.
+
+All LSST products that are built using :command:`scons` will execute Python tests using `pytest`_ and all tests should be written such that they are runnable using `pytest`_, a policy adopted in :jira:`RFC-69`.
```

Comment:
"and all tests should be ..." -> ", so all tests must be runnable using it."  We're trying to avoid mentioning RFCs directly in text.

Reply:
Yes. That text was there previously. I'll fix.